docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -1,54 +1,66 @@
 [[ingest-management-getting-started]]
-[chapter, role="xpack"]
-= Getting Started
+[role="xpack"]
+= Getting started
```

Comment:
I don't think these can be level one headings if they're no longer chapters.

Reply:
With that said, I think it's a good idea to get rid of the chapters.

Reply:
I'm planning to use level offsets to control topic nesting.

Reply:
Awesome!

Reply:
I am using asciidoctor magic. I might need to tweak it later on because I'm not getting the nesting depth I'd like to have, but this should work for us right now.